### DISCUSSION TITLE
Use of next/image not on Vercel but on a self-hosted server

### DISCUSSION THREAD

**Discussion by hk86** [vercel/next.js#22610] [Help] [answered]
Hi there,

I'm about to outsource some parts of my application, which are currently running on Vercel. Reason for this is the bandwidth usage I would like to reduce. I use the new next/image feature and I can call urls like https://www.example.com/_next/image?url=/image/cat.png&w=128&q=75 and I receive a compressed image. But when I deploy the application to my root server I can still call https://www.rootserver.com/_next/image?url=/image/cat.png&w=128&q=75 and I receive the image, but the image is not compressed. 

What do I miss? Do I need some special libraries installed on my root server?

Best regards,

HK

**Comment by leerob** [comment, marked as answer]
As long as you're using `next start`, Image Optimization will work out of the box.

https://nextjs.org/docs/basic-features/image-optimization#loader

**Reply by leerob** [reply]
If there is an issue, please file a bug report with a repro 🙏

**Reply by hk86** [reply]
Was this added with one of the recent nextjs versions? Currently I use 10.0.5

**Reply by leerob** [reply]
Nope, it's been like this since it was added!

**Reply by hk86** [reply]
Strange then I do not see a optimization on my root server:
Vercel: https://www.lolvvv.com/_next/image?url=/image/clash/clash-sm.png&w=256&q=75
Root Server: https://backend.lolvvv.com/_next/image?url=/image/clash/clash-sm.png&w=256&q=75

Looks like there is no image optimization library on my root server.

**Comment by hk86** [comment]
I resolved my issue. In case someone else might experience the same. The problem was that my nextjs was started by a systemd service. This service used a user and group, which is not allowed to write files or create folders. Since the optimized image is written to a cache https://github.com/vercel/next.js/blob/89ec21ed686dd79a5770b5c669abaff8f55d8fef/packages/next/next-server/server/image-optimizer.ts#L291 these permissions are required. After resolving the issue I see the optimized image!
